rfcs/038-sierra-series/README.md

```diff
@@ -0,0 +1,432 @@
+      }
+    ],
+    "type": "Series"
+  },
+  {   //773
+    "title": "Early European Books",
+    "type": "Work"
+  }
+]
+```
+
+- [3178769](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3178769?lang=eng&suite=cobalt&marcData=Y) 490 + 830 multiple entries + 773
+```yaml
+"partOf": [
+  {   // 490 & 830 with same information
+    "title": "Perspectives in Continental philosophy series; no. 39",
+    "workType": {
+      "id": "???",
+      "label": "Volumes",
+      "type": "Format"             
+    },
+    "partOf": [
+      {
+        "id": "abcdefg",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+              },
+            "value": "1089-3938",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Perspectives in Continental philosophy series",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {  // Second 830 with slighly different information
+    "title": "Fordham perspectives in continental philosophy",
+    "type": "Series"
+  },
+ {   // 830 & 773 contain the same information
+    "title": "ACLS Humanities E-Book.",
+    "type": "Work" //or Series?
+  }
+]
+```
+- [2125597](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2125597?lang=eng&suite=cobalt&marcData=Y)
+  490 with ISSN + 830 and 773. There are 1037 bibs that have the same ISSN in a series statement but
+  with different volume subfield.
+```yaml
+"partOf": [
+  {   // 490 +830 contain duplicated info
+    "partOf": [
+      {
+        "id": "abcdefg",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+            },
+            "value": "1064-3745",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Methods in Molecular Biology, Methods and Protocols",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {   // 773 
+    "title": "Springer eBooks",
+    "type": "Work"
+  }
+]
+```
+- [3001878](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3001878?lang=eng&suite=cobalt&marcData=Y)
+  Only 830 with no 490/440 or 773
+```yaml
+"partOf": [
+    {
+        "title": "Adamson Collection Wellcome Library", // subfield $a concatenated with $p
+        "type": "Series"
+    }
+]
+```
+- [1110225](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1110225?lang=eng&suite=cobalt&marcData=Y)
+Series statement in 440 with only the title
+```yaml
+"partOf": [
+    {
+        "title": "Monographs on inorganic and physical chemistry",
+        "type": "Series"
+    }
+]
+```
+- [1204561](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1204561?lang=eng&suite=cobalt&marcData=Y) 440 with id
+```yaml
+"partOf": [
+  {
+    "title": "Brill's studies in intellectual history; v. 13",
+    "workType": {
```

Comment:
Wouldn't use workType here. In any case it would be Journals, not Volumes. If we wanted to model journals more granularly it would either be with Items or another subtype of Work called Volume. Neither of which I think we want to do here, so it's probably better to collapse it and consider the ISSN non-unique.

Reply:
By "consider ISSN non unique" what do you mean? I think I see your point of not overthinking it and probably it's easier if we collapse the two, but I struggle to understand how we want to include the identifiers block then. So far (I think) we've always put the identifier on the thing it refers to but if we collapse the volume, we break that and the model becomes a bit misleading I think? We could decide to include the nested `partOf` only when we have an identifier and not if we don't have it?

Reply:
> Wouldn't use workType here. In any case it would be Journals, not Volumes. If we wanted to model journals more granularly it would either be with Items or another subtype of Work called Volume

I see. I think it would be a bit difficult in the transformer to figure out when `workType` `Volume` makes more sense than `Journal`. May e it would be best to leave the `workType` out always (even in the previous examples)? I'm not sure it's adding anything

Reply:
Yeah if we don't know for certain (ie it's a link to a work we actually have in the catalogue) then let's not try and add a workType.